k8s/perftest/common/ccd/data-store-api.yaml

```diff
@@ -24,6 +24,7 @@ spec:
       environment:
         CCD_DEFAULTPRINTURL: "http://ccd-case-print-service-perftest.service.core-compute-perftest.internal/jurisdictions/:jid/case-types/:ctid/cases/:cid"
         CCD_DM_DOMAIN: ^https?://(?:api-gateway\.preprod\.dm\.reform\.hmcts\.net|dm-store-perftest\.service\.core-compute-perftest\.internal(?::\d+)?)
+        DEFINITION_CACHE_JURISDICTION_TTL_SEC: 120
```

Comment:
please add a comment mentioning this should be removed when we move to v 1.3.1
Otherwise it'll become config mess

Reply:
@mario-paniccia done :)